Makefile

```diff
@@ -52,12 +52,16 @@ ocid: $(GO_SRC) | ${OCID_LINK}
 ocic: $(GO_SRC) | ${OCID_LINK}
 	$(GO) build -o $@ ./cmd/client/
 
+kpod: $(GO_SRC)
```

Comment:
Hm. Seemed to build without it on my machine, but it's not as if much of anything is vendored here - probably safest to include it. I'll update.